util/api/index.js

```diff
@@ -16,4 +16,4 @@ export const apiGetPendingCosmosMigration = (address, ethMigrationTxHash) => api
   `/migrate/pending/cosmos/${address}?eth_tx=${ethMigrationTxHash}`,
 );
 
-export const apiGetCosmosBalance = (address) => api.get(`/migrate/cosmos/${address}`);
+export const apiGetCosmosBalance = (address) => api.get(`/bank/balances/${address}`);
```

Comment:
Why this change?

Reply:
oh! actually this is not needed. Fixed here: https://github.com/likecoin/migrate-to-likecoin-chain/pull/57/commits/2665d6c41041aa2422b6d3186fc120e2be2c02f3